install/image-references

```diff
@@ -30,3 +30,27 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-gcp-machine-controllers:v4.0.0
+  - name: metal3-baremetal-operator
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-baremetal-operator:v4.2.0
+   - name: metal3-ironic
+    from:
```

Comment:
looks like a formatting error - from is not aligned with name above, same with the changes below

Reply:
Yes, "saw" it late. Tried to fix formatting errors in the latest version.